smtp/init.lua

```diff
@@ -167,15 +181,60 @@ curl_mt = {
                 header = header .. table.concat(opts.headers, '\r\n') .. '\r\n'
             end
 
-            local content_type = opts.content_type or 'text/plain'
-            local charset = opts.charset or 'UTF-8'
-            header = header .. 'Content-Type: ' .. content_type .. '; charset=' .. charset .. '\r\n'
+            local attachments = ''
+            if not opts.attachments or #opts.attachments == 0 then
+                local content_type = opts.content_type or 'text/plain'
+                local charset = opts.charset or 'UTF-8'
+                header = header .. 'Content-Type: ' .. content_type .. '; charset=' .. charset .. '\r\n'
+
+                if opts.subject then
+                    header = header .. 'Subject: ' .. opts.subject .. '\r\n'
+                end
+
+                body = header .. '\r\n' .. body
+            else
+                local MULTIPART_CONTENT_TYPE = 'Content-Type: multipart/mixed; boundary=MULTIPART-MIXED-BOUNDARY;\r\n'
```

Comment:
It would be good to mention exact standard you follow (at least because they're renewed from time to time).

Reply:
Resolved in PR #23.